hedera-node/hedera-app/src/main/java/com/hedera/node/app/state/merkle/MerkleSchemaRegistry.java

```diff
@@ -338,12 +342,19 @@ private RedefinedWritableStates applyStateDefinitions(
                                             md.stateDefinition().valueCodec());
                                     // MAX_IN_MEMORY_HASHES (ramToDiskThreshold) = 8388608
                                     // PREFER_DISK_BASED_INDICES = false
-                                    final var tableConfig = new MerkleDbTableConfig((short) 1, DigestType.SHA_384)
+                                    final MerkleDbConfig merkleDbConfig =
+                                            platformConfiguration.getConfigData(MerkleDbConfig.class);
+                                    final var tableConfig = new MerkleDbTableConfig(
+                                                    (short) 1,
+                                                    DigestType.SHA_384,
+                                                    merkleDbConfig.maxNumOfKeys(),
+                                                    merkleDbConfig.hashesRamToDiskThreshold())
                                             .maxNumberOfKeys(def.maxKeysHint());
```

Comment:
This line is now redundant

Reply:
Please see: https://github.com/hashgraph/hedera-services/pull/15562/commits/e1fd219ac902a0cc4e45b19bc1b57e7245b63305